rimage/depth_map_raw.go

```diff
@@ -88,20 +91,25 @@ func readDepthMapRaw(ff io.Reader, firstBytes int64) (*DepthMap, error) {
 func readDepthMapViam(ff io.Reader) (*DepthMap, error) {
 	f := bufio.NewReader(ff)
 	dm := &DepthMap{}
+	data := make([]byte, 8)
 
-	rawWidth, err := _readNext(f)
```

Comment:
_readNext is coded to read in LittleEndian byte order, so for backwards compatbility with .dat and .dat.gz files, we can keep it, but replace the Viam Depth type to read in BigEndian order